thrifty-schema/src/main/java/com/microsoft/thrifty/schema/AbstractUserElementBuilder.java

```diff
@@ -64,5 +65,16 @@ public TBuilder annotations(Map<String, String> annotations) {
         return (TBuilder) this;
     }
 
+    public TBuilder type(TElement type) {
```

Comment:
Agreed it could be misused, but then again so could the others I think (especially since this is just a convenience setter over existing ones)